index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Моя первая страница</h1>
+    <p><a href="../if-html/about/index.html">Об авторе</a></p>
```

Comment:
Seems the value for href attribut is wrong.
```suggestion
    <p><a href="./about/index.html">Об авторе</a></p>
```

Reply:
fixed